employees/common/exports.py

```diff
@@ -0,0 +1,140 @@
+        cell = worksheet.cell(row=ExcelGeneratorSettingsConstants.HEADERS_ROW.value, column=col_num)
+        cell.value = column_title
+        set_format_styles_for_main_cells(cell)
+        width_settings = ExcelGeneratorSettingsConstants.COLUMNS_WIDTH_FOR_SINGLE_USER.value
+        set_columns_width(worksheet, col_num, width_settings)
+
+
+def fill_data_in_reports_for_single_user(worksheet, reports):
+    row_number = ExcelGeneratorSettingsConstants.FIRST_ROW_FOR_DATA.value
+    for report in reports:
+        data_in_row = [
+            report.date,
+            report.project.name,
+            str(report.task_activities),
+            report.work_hours_str,
+            report.description,
+        ]
+        for col_num, cell_value in enumerate(data_in_row, 1):
+            cell = worksheet.cell(row=row_number, column=col_num)
+            if col_num == ExcelGeneratorSettingsConstants.DESCRIPTION_COLUMN_FOR_SINGLE_USER.value:
+                wrapped_alignment = Alignment(
+                    vertical=ExcelGeneratorSettingsConstants.VERCTICAL_TOP.value, wrap_text=True
+                )
+                cell.alignment = wrapped_alignment
+                cell.value = cell_value
+            elif col_num == ExcelGeneratorSettingsConstants.HOURS_COLUMN_FOR_SINGLE_USER.value:
+                cell.value = ExcelGeneratorSettingsConstants.TIMEVALUE_FORMULA.value.format(cell_value)
+                cell.number_format = ExcelGeneratorSettingsConstants.HOURS_FORMAT.value
+            else:
+                cell.value = cell_value
+        row_number += 1
+    return row_number
+
+
+def summarizing_reports(worksheet, last_row):
+    total_cell = worksheet.cell(row=last_row, column=ExcelGeneratorSettingsConstants.TOTAL_COLUMN.value)
+    total_cell.value = ExcelGeneratorSettingsConstants.TOTAL.value
+    set_format_styles_for_main_cells(total_cell)
+    total_hours_cell = worksheet.cell(
+        row=last_row, column=ExcelGeneratorSettingsConstants.HOURS_COLUMN_FOR_SINGLE_USER_REPORTS.value
+    )
+    total_hours_cell.value = ExcelGeneratorSettingsConstants.TOTAL_HOURS_FORMULA_FOR_SINGLE_USER.value.format(
+        last_row - 1
+    )
+    total_hours_cell.number_format = ExcelGeneratorSettingsConstants.TOTAL_HOURS_FORMAT.value
+    set_format_styles_for_main_cells(total_hours_cell)
+
+
+def generate_xlsx_for_single_user(author: CustomUser):
+    reports = author.get_reports_created()
+    workbook = Workbook()
+    worksheet = workbook.active
+    worksheet.title = f"{author.first_name} {author.last_name}"
+    fill_headers_for_single_user(worksheet)
+    last_row = fill_data_in_reports_for_single_user(worksheet, reports)
+    summarizing_reports(worksheet, last_row)
+    return workbook
+
+
+def generate_xlsx_for_project(project: Project):
+    authors = project.members.all()
+    workbook = Workbook()
+    for author in authors:
+        worksheet = workbook.active
```

Comment:
Those 6 lines above are quite important and reused in two methods (here and above). How about creating a separate method out of them?

Reply:
I created separate method to set title of active worksheet, I think I couldn't give there more stuff because generate xlsx for project and single user are different and filters will be different but maybe I made something wrong.